cartography/intel/aws/sqs.py

```diff
@@ -0,0 +1,140 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_sqs_queue_list(boto3_session: boto3.session.Session, region: str) -> List[str]:
+    client = boto3_session.client('sqs', region_name=region)
+    paginator = client.get_paginator('list_queues')
+    queues: List[Any] = []
+    for page in paginator.paginate():
+        queues.extend(page.get('QueueUrls', []))
+    return queues
+
+
+@timeit
+def get_sqs_queue_attributes(
```

Comment:
Similarly, below `@timeit` and before the function definition, add `@aws_handle_regions`.